src/lambda/handler-runner/docker-runner/DockerRuntime.js

```diff
@@ -0,0 +1,33 @@
+import { supportedRuntimesArquitecture } from "../../../config/supportedRuntimes.js"
```

Comment:
@BenjaminBergerM I think it will cause error since on config you name it right as `architecture`, maybe convert all "architecture" to `architecture`?